src/components/Company.js

```diff
@@ -1,38 +1,39 @@
 import React from "react";
 import Peers from "./Peers";
 import { useSelector } from "react-redux";
+import { Loading } from "./Loading";
 
 export const Company = () => {
-  const state = useSelector(state => state);
-  const companyOverview = state.selectedCompanyOverview;
+  const companyOverview = useSelector(state => state.selectedCompanyOverview);
+
+  const renderCompanyComponent = () => (
+    <>
+      <div className="company-overview__title">
+        {companyOverview.companyName} ({companyOverview.symbol})
+      </div>
+      <div>
+        <a
+          target="_blank"
+          rel="noopener noreferrer"
+          className="company-overview__website"
+          href={`${companyOverview.website}`}
+        >
+          {companyOverview.website}
+        </a>{" "}
+      </div>
+      <div className="company-overview__text">
+        <p>{companyOverview.description}</p>
+      </div>
+    </>
+  );
   return (
```

Comment:
New line between the `);` and `return (`, please! Makes it hard to visually break up when reviewing, otherwise.